docs/source/developers/guide/tutorials/r_tutorial.rst

```diff
@@ -22,21 +22,429 @@
 
 .. _r-tutorial:
 
-**********
-R tutorial
-**********
+***********
+R tutorials
+***********
+
+
+Writing Bindings Walkthrough
+============================
 
 The first R package tutorial to be included in the New Contributor's
 guide is a **Walkthrough** added in the **Writing Bindings**
 vignette. With time we will try to include additional tutorials
 directly into this guide.
 
-Writing Bindings Walkthrough
-----------------------------
-
 This tutorial will show how to do a binding of a C++ function
 `starts_with() <https://arrow.apache.org/docs/cpp/compute.html#containment-tests>`_
 to the (base) R function ``startsWith()``.
 
 To view the tutorial follow the
 `Walkthrough section of the Writing Bindings article <https://arrow.apache.org/docs/r/articles/developers/bindings.html#walkthrough>`_.
+
+
+R tutorial on adding a lubridate binding
+========================================
+
+In this tutorial we will document a simple binding contribution
+to Arrow R package following the steps specified by the
+:ref:`quick-ref-guide` section of the guide and a more detailed
+:ref:`step_by_step` section. Navigate there whenever there is
+some information you may find is missing here.
+
+The binding will be added to the ``expression.R`` file in the 
+R package. But you can also follow these steps in case you are
+adding a binding that will live somewhere else.
+
+.. seealso::
+   
+   To read more about the philosophy behind R bindings, refer to the
+   `Writing Bindings article <https://arrow.apache.org/docs/r/articles/developers/bindings.html>`_.
+
+This tutorial is different from the :ref:`step_by_step` as we
+will be working on a specific case. This tutorial is not meant
+as a step-by-step guide.
+
+**Let's start!**
+
+Set up
+------
+
+Let's set up the Arrow repository. We presume here that Git is
+already installed. Otherwise please see the :ref:`set-up` section.
+
+Once the `Apache Arrow repository <https://github.com/apache/arrow>`_
+is forked we will clone it and add the link of the main repository
```

Comment:
Do we instruct people on how to fork it?  If so, can we link to that.  If not, we should explicitly tell the reader that they'll need to fork it.

Reply:
We have the instructions in the guide, I will add a link to it 👍